patched-fonts/0xProto/README.md

```diff
@@ -0,0 +1,42 @@
+# 0xProto
```

Comment:
This file and all others in `patched-fonts/` is autogenerated and should not be added manually.
* This README
* all font files

Reply:
Understood! Removed all content from `patched-fonts/`